docs/integrations/api.md

```diff
@@ -0,0 +1,553 @@
+# API
+
+Here you can find various Lido APIs which you can integrate in your app or website:
+
+## Lido APR
+
+API provides Ethereum and Lido staking APR, which include:
+
+### Last Lido APR for stETH
+
+The latest staking APR value that is retrieved from our database. For Lido V1, we collected APR values by periodically fetching [oracle report events](https://docs.lido.fi/contracts/lido-oracle#receiver-function-to-be-invoked-on-report-pushes). For the V2 version, the value is calculated based on [rebase events](https://github.com/lidofinance/lido-dao/blob/e9509d77f010fec76899e25ccde785c8de47bd42/contracts/0.4.24/Lido.sol#L232).
+
+
+V1 APR calculation: 
+
+```
+protocolAPR = (postTotalPooledEther - preTotalPooledEther) * secondsInYear / (preTotalPooledEther * timeElapsed)
+lidoFeeAsFraction = lidoFee / basisPoint
+userAPR = protocolAPR * (1 - lidoFeeAsFraction)
+```
+
+V2 APR calculation:
+
+```
+// Emits when token rebased (total supply and/or total shares were changed)
+event TokenRebased(
+    uint256 indexed reportTimestamp,
+    uint256 timeElapsed,
+    uint256 preTotalShares,
+    uint256 preTotalEther, /* preTotalPooledEther */
+    uint256 postTotalShares,
+    uint256 postTotalEther, /* postTotalPooledEther */
+    uint256 sharesMintedAsFees /* fee part included in `postTotalShares` */
+);
+
+preShareRate = preTotalEther * 1e27 / preTotalShares
+postShareRate = postTotalEther * 1e27 / postTotalShares
+
+userAPR = 
+    secondsInYear * (
+        (postShareRate - preShareRate) / preShareRate
+    ) / timeElapsed
+```
+
+https://eth-api.lido.fi/v1/protocol/steth/apr/last
+
+Response body:
+
+```json
+{
+  "data": {
+    // Block timestamp in seconds
+    "timeUnix": 1682425475,
+    // APR value
+    "apr": 4.77
+  },
+  // token details
+  "meta": {
+    "symbol": "stETH",
+    "address": "0xae7ab96520DE3A18E5e111B5EaAb095312D7fE84",
+    "chainId": 1
+  }
+}
+```
+
+### **Simple Moving Average Lido APR for 7 last days:**
```

Comment:
Perhaps this should be put above. This is the calculation we want to use in integrations and I think it's worth highlighting somehow